TACC/Core-CMS-Resources
# FP-1226: Add a2cps favicon

## Required By

- https://github.com/TACC/Core-CMS/pull/380

## Overview

Add a2cps favicon.

## Issues

- [FP-1226](https://jira.tacc.utexas.edu/browse/FP-1226)

## Changes

- Add favicon resource.
- Set path to new resource as favicon URL.

## Screenshots

![FP-1226 A2CPS Favicon](https://user-images.githubusercontent.com/62723358/137366027-4ba50a6a-8d1c-4da8-99e8-7e669f57db6e.png)

## Testing

1. Know how to [Test CMS Changes for a Custom Project](https://github.com/TACC/Core-CMS-Resources/wiki/Test-New-CMS-Site).
2. Set up testing for `a2cps-cms` on Core-CMS branch `task/FP-1226-a2cps-favicon`.
3. Confirm that favicon is visible in browser tab/bar.

<details><summary>Debugging</summary>

_If favicon is not immediately visible, search live markup for "favicon" and make sure the relevant `<link ...>` has an `href` with a URL that has `a2cps`. If not, confirm your environment is setup accurately. If so, poke me and we'll figure it out._

</details>

## Caveats

1. <details>
	<summary>(Inaccurate) Portal Favicon</summary>

	~~If user visits Portal before CMS, default Portal favicon will be cached by user's browser instead of this CMS one. There is no PR to change favicon for Portal, because the Portal does not appear to support custom favicon without adding file manually to server, and manually editing `settings_custom.py`. I do not have access to do this.~~

	</details>
